plugins/modules/pacemaker_stonith.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2025, Dexter Le <dextersydney2001@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: pacemaker_stonith
+short_description: Manage pacemaker stonith
+author:
+  - Dexter Le (@munchtoast)
+version_added: 10.8.0
+description:
+  - This module can manage stonith in a Pacemaker cluster using the pacemaker CLI.
```

Comment:
Not familiar with "stonith" but it looks like a name of a product/service/something. Shouldn't it be capitalized throughout the docs?

Reply:
If it's about the technique (https://en.wikipedia.org/wiki/STONITH#STONITH), it should be written "STONITH".